tests/wpt/mozilla/tests/css/inline_stacking_context.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Inline stacking context positioned correctly</title>
+        <link rel=match href="/_mozilla/css/inline_stacking_context_ref.html">
```

Comment:
create-wpt forms the path like this, so I guess it should be modified to to understand relative URLs.